core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceSpec.scala

```diff
@@ -243,7 +243,17 @@ class BillingApiServiceSpec extends ApiServiceSpec with MockitoSugar {
   }
 
   it should "return 400 when creating a project with enableFlowLogs but not highSecurityNetwork" in withTestDataApiServices { services =>
-    Post("/billing", CreateRawlsBillingProjectFullRequest(RawlsBillingProjectName("test_good"), services.gcsDAO.accessibleBillingAccountName, highSecurityNetwork = Some(false), enableFlowLogs = Some(true), None)) ~>
+    Post("/billing", CreateRawlsBillingProjectFullRequest(RawlsBillingProjectName("test_good"), services.gcsDAO.accessibleBillingAccountName, highSecurityNetwork = Some(false), enableFlowLogs = Some(true), None, None)) ~>
+      sealRoute(services.billingRoutes) ~>
+      check {
+        assertResult(StatusCodes.BadRequest) {
+          status
+        }
+      }
+  }
+
+  it should "return 400 when creating a project with privateIpGoogleAccess but not highSecurityNetwork" in withTestDataApiServices { services =>
+    Post("/billing", CreateRawlsBillingProjectFullRequest(RawlsBillingProjectName("test_good"), services.gcsDAO.accessibleBillingAccountName, highSecurityNetwork = Some(false), None, privateIpGoogleAccess = Some(true), None)) ~>
```

Comment:
Should we have a positive test where we enable `privateGoogleAccess`?  I know it doesn't really do anything in tests since we aren't calling Google, so this is actually just a question.  If there's really no value in it, don't bother writing the test.

Reply:
Not sure how much value it added, but I added in some positive tests for enabling `privateIpGoogleAccess` along with high security networks and flow logs